<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeetingRoomDaily</class>
 <widget class="QWidget" name="MeetingRoomDaily">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aula riunioni / Aula corsi</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>672</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="gridSize">
    <size>
     <width>431</width>
     <height>21</height>
    </size>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>689</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="riunioniButton">
      <property name="text">
       <string>Aula riunioni</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="corsiButton">
      <property name="text">
       <string>Aula corsi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
